Posts/2021/09/Understanding Get-ACL and AD Drive Output.md

```diff
@@ -0,0 +1,245 @@
+---
+post_title: Understanding Get-ACL and AD Drive Output
+username: farisnt@gmail.com
+Categories: PowerShell
+tags: PowerShell, Active Directory, ACL
+Summary: Understanding Get-ACL and AD Drive Output
+---
+
+## Understanding `Get-ACL` and AD: Drive Output
+
+Understanding Active Directory ACL using PowerShell can be a bit tricky.
+There are no out-of-the-box cmdlets with ActiveDirectory PowerShell module to help in settings the permission quickly.
+While there are no cmdlets, you can nevertheless manage AD permissions using the AD PowerShell drive.
+
+In this post, I will try to simplify Active Directory ACL and how to read the result easily, So let's start.
+
+## Prerequisites
+
+To follow along with this article, you need the following:
+
+- PowerShell 7. x or Windows PowerShell 5.1
+- The user is memeber of Domain Admin AD Group.
+- Windows Server 2012, 2016, 2019 or 2022 with Active Directory Domain Service role installed.
+
+The domain name used for this tutorial is **Contoso.com**.
+
+## Reading Active Directory Permission using `Get-ACL`
+
+Reading Active Directory permission using `Get-ACL` doesn't require a long line of code.
+However, we are reading from AD and not the  FileSystem provider. So we use the **AQ:** drive. This drive is automatically loaded when you load the ActiveDirectory module.
+
+>[!NOTE]
+>To import Active Directory Module, use the **Import-Module ActiveDirectory**
+
+Also, the path format used to query objects is the **Distinguished Name** for the AD object.
```

Comment:
You could phrase this better, IMHO:

When you query for an object to get its ACL, you need to search based on **Distinguished Name**.